Esri/arcgis-maps-sdk-swift-samples
# [New] Set up location-driven geotriggers

## Description

This PR implements `Set up location-driven geotriggers` in `Display information` category.
URL to README: https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-SetUpLocationDrivenGeotrigers/Shared/Samples/Set%20up%20location-driven%20geotriggers

## Linked Issue(s)

- `swift/issues/4135`

## How To Test

* Watch the location display move and the status texts change accordingly.
* Tap the "Current Section" button to see the current section information when available.
* Tap the "Point of Interest" button to see the point of interest information when available.

## Screenshots

![set-up-location-driven-geotriggers](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/90805879-b3de-464a-968f-3425272c3f45)
